app/create_file.py

```diff
@@ -1 +1,46 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+import os
+
+
+def create_dir_or_file(dir_l: list, file_l: list) -> None:
+    dir_str = os.path.join("app", *dir_l[1:])
+    os.makedirs(dir_str) if dir_l[1:] else None
```

Comment:
add exist_ok to makedirs to avoid errors
```suggestion
    os.makedirs(dir_str, exist_ok=True) if dir_l[1:] else None
```